docs/2.5. APIs - Query Parameters.md

```diff
@@ -116,27 +125,31 @@ Payload Resources
 ```
 
 ***Notes***
-* In this case the server has accepted the client's paging size limit request. If the client had requested a page size which the server was unable to honour, the actual page size used would be returned in X-Paging-Limit.
+
+- In this case the server has accepted the client's paging size limit request. If the client had requested a page size which the server was unable to honour, the actual page size used would be returned in X-Paging-Limit.
 
 **Example 3: Request With Since Parameter**
 
 ***Request***
-```
+
+```http
 GET /x-nmos/query/v1.1/nodes?paging.since=0:4
 ```
 
 ***Response***
 
 Headers
-```
+
+```http
 Link: <http://example.api.com/x-nmos/query/v1.1/nodes/?paging.since=0:14&paging.limit=10>; rel="next", <http://example.api.com/x-nmos/query/v1.1/nodes/?paging.until=0:4&paging.limit=10>; rel="prev"
 X-Paging-Limit: 10
 X-Paging-Since: 0:4
 X-Paging-Until: 0:14
 ```
 
 Payload Resources
-```
+
+```http
```

Comment:
This should be `json` or whatever is consistent with the array response examples above.